### ISSUE TITLE
Feature wish: TapRecognizer: Direct firing if tapCount is reached

### ISSUE THREAD

**Issue by FFittkau** [hammerjs/hammer.js#747]
I would like to have an option for TapRecognizer where it directly fires if the tapCount is reached (if no requireFailure is defined of course)

I am using double recognition code from the website and done a logging for double tap:

First column is passed ms since first event:
0 input: mousedown
74ms input: mouseup
145ms input: mousedown
218ms input: mouseup
520ms double tap

So it is waiting the 300ms interval after the second tap. Since I am building a highly reactive web app, I would like it to fire instantly after tapCount == 2.

Longer text on stackoverflow:
http://stackoverflow.com/questions/27780185/double-tap-recognition-takes-too-long-hammer-js-2-0

**Comment by jareware** [comment]
I have the exact same issue as you; did you manage to work this into your solution somehow @FFittkau? Wondering if http://stackoverflow.com/a/27974264 is the way to go...

**Comment by FFittkau** [comment]
Yes http://stackoverflow.com/a/27974264 is my solution. The "if (!this.hasRequireFailures())" seems to be brocken somehow... Therefore, you have to implement your own DblTapHandler to work around the bug. I could upload my hammer.js if you want it.

**Comment by jareware** [comment]
Having your DblTapHandler implementation linked here (as a gist or something) might be good for anyone else who ends up hitting this issue!

**Comment by FFittkau** [comment]
Can be found at:
https://gist.github.com/FFittkau/462df729e728ffb3188e#file-gistfile1-js

**Comment by jareware** [comment]
Thanks, appreciated! :)

**Comment by runspired** [comment]
I'd be interested in pursuing a fix for this in Hammer itself if the issue is indeed a broken behavior around `hasRequireFailures`.

**Comment by FFittkau** [comment]
Could also been the way hasRequiredFailures works but having more clicks two (a double click) did not make any sense in my scenario. Therefore, an extra DoubleClickHandler - like in the gist - might make sense.

**Comment by llamerr** [comment]
I thought this is what hammer-time does, it's strange it doesn't work this way from the box.
